capability.go

```diff
@@ -0,0 +1,88 @@
+package stripe
+
+import "encoding/json"
+
+// CapabilityDisabledReason describes why a capability is disabled.
+type CapabilityDisabledReason string
+
+// List of values that CapabilityDisabledReason can take.
+const (
+	CapabilityDisabledReasonPendingOnboarding        CapabilityDisabledReason = "pending.onboarding"
+	CapabilityDisabledReasonPendingReview            CapabilityDisabledReason = "pending.review"
+	CapabilityDisabledReasonRejectedFraud            CapabilityDisabledReason = "rejected_fraud"
```

Comment:
Oof, it looks like this isn't a mistake (as it's in the API), but we use an underscore in `rejected_fraud` and a dot everywhere else (i.e. `rejected.listed`)?

Think it's okay to merge it given it's ubiquitous, but will follow up elsewhere.